java/testing/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+list(APPEND javaf3dTests_list
+     ${CMAKE_CURRENT_SOURCE_DIR}/TestJavaBindings.java
+    )
+
+set(java_test_args "-ea") # enable assertions
+if(APPLE)
+  # On macOS, this argument is mandatory in order to create native window
+  list(APPEND java_test_args "-XstartOnFirstThread")
+endif()
+
+# On macOS, VTK 9.1 is crashing in modifyDPIForBackingScaleFactorOfWindow, see:
+# https://gitlab.kitware.com/vtk/vtk/-/merge_requests/6845 (introduces the crash)
+# https://gitlab.kitware.com/vtk/vtk/-/merge_requests/9372 (fixes the crash)
+# Disabling the tests if we are between these two commits
+set(java_test_disabled OFF)
+if(APPLE AND VTK_VERSION VERSION_GREATER 9.0.0 AND VTK_VERSION VERSION_LESS_EQUAL 9.1.20220617)
```

Comment:
so vtk 9.0.1 has the issue ?

Reply:
It's funny enough, the commit producing the crash is the first one after the `9.0.0` tag

Reply:
lol